plugins/modules/storage/pmem/pmem.py

```diff
@@ -0,0 +1,457 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2022, Masayoshi Mizuma <msys.mizuma@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+author:
+ - Masayoshi Mizuma (@mizumm)
+module: pmem
+short_description: Configure Intel Optane Persistent Memory modules.
+description:
+ - This module allows Configuring Intel Optane Persistent Memory modules
+   (PMem) using ipmctl and ndctl command line tools.
+requirements:
+ - ipmctl and ndctl command line tools.
+ - xmltodict
+options:
+  AppDirect:
+    description:
+     - Percentage of the total capacity to use in AppDirect Mode (0-100).
```

Comment:
```suggestion
     - Percentage of the total capacity to use in AppDirect Mode (C(0)-C(100)).
```
Out of curiosity: are fractional percentages also allowed?